2021/32xxx/CVE-2021-32809.json

```diff
@@ -1,18 +1,88 @@
 {
```

Comment:
'CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:L/I:L/A:N' does not match '^(ftp|http)s?://\\S+$'